tests/test_ma_functions.py

```diff
@@ -0,0 +1,70 @@
+import numpy.ma as ma
+from numpy.ma.testutils import assert_equal
+
+from foapy.ma_alphabet import ma_alphabet
+
+
+class TestMaFunctions:
```

Comment:
```suggestion
class TestMaAlphabet(TestCase):
```

Example https://realpython.com/pytest-python-testing/#less-boilerplate